README.md

```diff
@@ -148,16 +151,36 @@ lib.languages {
 }
 ```
 
-### Feature toggle "FrontendBaseUrlInPagePreview"
+### Available features toggles
 
 To change the setting for this extension feature either use Localconfiguration.php: or AdditionalConfiguration.php:
 
+**headless.frontendUrls** or **~~FrontendBaseUrlInPagePreview~~** (deprecated)
+
+This feature toggle extends current SiteConfiguration (and it's variants) with new field for Frontend Url
+(url frontend of PWA app). This new field is used when there is a need to preview a page such as: "view" module or right click on a page + show, or the 'eye' icon in page view
+& allow generating proper cross-domain links for headless instance.
 ```
-$GLOBALS['TYPO3_CONF_VARS']['SYS']['features']['FrontendBaseUrlInPagePreview'] = true;
```

Comment:
should we also document that  the old value was this, and it changed in some version (or add a section below about upgrading headless between versions)

Reply:
@tmotyl Yes. I planned to add UPGRADE info docs. But this changes are good start to do it. I updated commit with UPGRADE.md file.